### PR TITLE
Update register-windows-admin-center.md

### PR DESCRIPTION
**Pull request by SharmaKartikay**
The added note is crucial as most customers are running into the issue with Proxy settings and opening support cases.

### CODE REVIEW

**Comment by PRMerger3** [comment]
@SharmaKartikay : Thanks for your contribution! The author(s) have been notified to review your proposed change. @prasidharora

File: azure-stack/hci/manage/register-windows-admin-center.md [hunk 1]
```diff
@@ -16,6 +16,9 @@ To use Azure services with Windows Admin Center, you must first [install Windows
    > [!IMPORTANT]
    > In order to use Windows Admin Center to register Azure Stack HCI clusters, your Windows Admin Center gateway must be registered to an Azure Active Directory (Azure AD) application ID that is approved by your organization's Azure AD admin. Register Windows Admin Center on the same management PC you plan to use to register your cluster(s), using the same Azure Active Directory (tenant) ID.
 
+   > [NOTE]
+   > If your network is confogured to use a proxy server for internet connection, it is required to configire the proxy settings on Windows Admin Center Gateway Settings. After configuring the proxy restart ServerManagmentGateway (Windows Admin Center) service on the server installed with Windows Admin Center before proceeding with Azure AD registration, else the registration may fail.
+
```
Comment (sethmanheim) [inline azure-stack/hci/manage/register-windows-admin-center.md:21]: Please fix typo "configire"

**Comment by PRMerger13** [comment]
@SharmaKartikay : Thanks for your contribution! The author(s) have been notified to review your proposed change. @prasidharora

File: azure-stack/hci/manage/register-windows-admin-center.md (hunk 1 above)
Comment (SharmaKartikay) [reply azure-stack/hci/manage/register-windows-admin-center.md:21]: Fixed the typo and made some changes for ease of understanding.

Comment (sethmanheim) [reply azure-stack/hci/manage/register-windows-admin-center.md:21]: The format of the note section is incorrect -- it needs to be [!NOTE].

**Comment by PRMerger-2** [comment]
@SharmaKartikay : Thanks for your contribution! The author(s) have been notified to review your proposed change. @prasidharora

File: azure-stack/hci/manage/register-windows-admin-center.md (hunk 1 above)
Comment (SharmaKartikay) [reply azure-stack/hci/manage/register-windows-admin-center.md:21]: Please check its corrected now.

**Comment by sethmanheim** [comment]
Thanks @SharmaKartikay! 
#sign-off

**Comment by PRMerger16** [comment]
@sethmanheim: I'm sorry - only the author of this article, @prasidharora, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by sethmanheim** [comment]
@prasidharora Can you please sign off?

**Comment by sethmanheim** [comment]
@SharmaKartikay Not sure why this article has the listed author -- so in a separate PR in the private repo, I updated the author's name to myself and included your change above. It should be live after 3 PM PST today, so I am closing this PR. Thanks again for the submission! #please-close

**Pull request closed without merging**